dev/tests/integration/testsuite/Magento/Newsletter/Controller/ManageTest.php

```diff
@@ -84,7 +85,7 @@ public function testSaveActionRemoveSubscription()
          * Check that success message
          */
         $this->assertSessionMessages(
-            $this->equalTo(['We removed the subscription.']),
+            $this->equalTo(['We updated the subscription.']),
```

Comment:
We would need to update this message to related to the changes to the messages I have suggested.